packages/openapi-typescript/src/utils.ts

```diff
@@ -55,13 +55,13 @@ export function getSchemaObjectComment(v: CommentObject, indentLv?: number): str
   const output: string[] = [];
 
   // * Not JSDOC tags: [title, format]
-  if (v.title) output.push(`${v.title} `);
-  if (v.summary) output.push(`${v.summary} `);
-  if (v.format) output.push(`Format: ${v.format} `);
+  if (v.title) output.push(v.title);
+  if (v.summary) output.push(v.summary);
+  if (v.format) output.push(`Format: ${v.format}`);
```

Comment:
The `comment` function already does the job of adding a trailing space in case of a single-line comment